pybamm/input/parameters/lead-acid/experiments/1C_discharge_from_full/parameters.csv

```diff
@@ -5,8 +5,11 @@ Name [units],Value,Reference,Notes
 Reference temperature [K],294.85,Room temperature,
 Maximum temperature [K],333.15,,
 Ambient temperature [K], 294.85,,
-Heat transfer coefficient [W.m-2.K-1],10,,
-Initial temperature [K],294.85,Room temperature,
+Negative current collector surface heat transfer coefficient [W.m-2.K-1],0,,
```

Comment:
is there a reason the defaults aren't just uniform cooling from all surfaces (I think this might be what people would expect?)

Reply:
I thought people would be more expecting the model to represent a single layer in the center of a cell as opposed to a single layer being cooled from each surface, since this is never really seen. 

I really don't mind what the defaults are though, so feel free to change them to whatever you think is clearest

Reply:
ah ok. I was thinking of an idealized single layer cell that might be used e.g. in a lab. Not sure I have a strong preference, but I think most people will expect to see a single value for h by default